Stripe/STPAddressFieldTableViewCell.m

```diff
@@ -197,6 +205,19 @@ - (NSString *)captionForAddressField:(STPAddressFieldType)addressFieldType {
     }
 }
 
+- (void)delegateCountryCodeDidChange {
+    
+    NSString *countryCode = nil;
+    if ([self.delegate respondsToSelector:@selector(addressFieldTableViewCountryCode)]) {
```

Comment:
can we just pass in the country code here, instead of getting it from the delegate?